lib/message_quickly/callback_parser.rb

```diff
@@ -55,9 +56,13 @@ def initialize(json)
     def parse
       events = []
       process_entry_json(@json['entry']) do |params|
-        WEBHOOK_LOOKUP.keys.each do |key|
-          if params[:messaging][key]
-            events << WEBHOOK_LOOKUP[key].new(params[:messaging])
+        if params[:changes].present?
```

Comment:
This could potentially be:

```
if params.dig(:changes)
```

but since we already have a dependency on Rails, I don't mind using the ActiveSupport methods, as the intent reads clearer.